.prettierrc

```diff
@@ -0,0 +1,26 @@
+{
+    "useTabs": false,
+    "endOfLine": "lf",
+    "printWidth": 88,
+    "trailingComma": "none",
+    "overrides": [
+        {
+            "files": ["*.js"],
+            "options": {
+                "tabWidth": 4
+            }
+        },
+        {
+            "files": ["*.html", "*.json", "*.prettierrc", "*.css"],
+            "options": {
+                "tabWidth": 4
+            }
+        },
```

Comment:
why those are separate? shouldnt they be one like:
```
["*.js", "*.html", "*.json", "*.prettierrc", "*.css"]
```
since options are the same